src/QieyunEncoder/_音位配列規則表.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+
+from enum import IntEnum
+
+
+class 合法性等級(IntEnum):
+    無效, 強非法, 弱非法, 弱合法, 稀有合法, 強合法 = range(6)
+
+    @property
+    def 字符串(self) -> str:
+        return self.name
+
+
+# 以下規則只適用於更優音韻地位
+# 每條規則是三元組 (合法性等級, 禁止的情況, 排除的情況)
+# 在規則下進行簡要解釋，並舉出違反規則的例外小韻（根據數據中對一些爭議小韻的處理不同，例外小韻可能有差異）
+音位配列規則表 = [
```

Comment:
Use `tuple` instead of `list`

```suggestion
音位配列規則表 = (
```

Reply:
> Use `tuple` instead of `list`

What is it for? :thinking:

I think a tuple here doesn't make sence _semantically_. It's more of a `list[tuple[合法性等級, str, Optional[str]]]` than a `tuple[Record, Record, Record, (AgainTheSame)Record...`.

Or is it for _immutability_ I guess? (which doesn't make sence to me either because Python doesn't work that way (?)

Reply:
Taken from <https://stackoverflow.com/a/1708538>:

- Tuples are faster than lists. If you're defining a constant set of values and all you're ever going to do with it is iterate through it, use a tuple instead of a list.
- It makes your code safer if you “write-protect” data that does not need to be changed. Using a tuple instead of a list is like having an implied assert statement that this data is constant, and that special thought (and a specific function) is required to override that.
- Some tuples can be used as dictionary keys (specifically, tuples that contain immutable values like strings, numbers, and other tuples). Lists can never be used as dictionary keys, because lists are not immutable.

Reply:
Oh, so it _is_ resonable in fact.

And also I've just found that there's even a `tuple[T, ...]` syntax for annotating a tuple to be used as a homogeneous, arbitrary-length containers.

So Pythonic :joy::joy:

(or maybe I'm just poisoned too much by tuples in Rust, TypeScript etc. :see_no_evil: